source/MRMesh/MRColorMapAggregator.h

```diff
@@ -0,0 +1,160 @@
+#pragma once
+#include "MRMeshFwd.h"
+#include "MRVector.h"
+#include "MRColor.h"
+#include "MRId.h"
+#include "MRBitSet.h"
+
+
+namespace MR
+{
+
+template<typename Tag>
+class ColorMapAggregator
+{
+public:
+    using ColorMap = Vector<Color, Id<Tag>>;
+    using ElementsBitSet = TaggedBitSet<Tag>;
+
+    ColorMapAggregator() = default;
+
+    void setDefaultColor( const Color& color );
+
+    void setColorMap( int i, const ColorMap& colorMap, const ElementsBitSet& elementsBitSet );
+
+    void resetColorMap( int i );
```

Comment:
Pls add comments to all public functions, also I think we need functions to insert or remove colormap to some index

Reply:
fixed